### PR TITLE
Fix filepaths in useful diagnostic section for logs

### PR DESCRIPTION
**Pull request by sarcasticadmin**
## Description

I was trying to diagnose an error on my mark 1 and noticed that the filepaths were incorrect in the **useful diagnostic** section. Updated the paths to be the correct paths and now I get the output I was expecting (included below)

## Previous Behavior
```
root@mark_1:~# cat /var/log/mycroft-skills.log | grep -i error | grep -i picker
cat: /var/log/mycroft-skills.log: No such file or directory
```

```
root@mark_1:~# tail -f /var/log/skills.log
tail: cannot open _/var/log/skills.log_ for reading: No such file or directory
tail: no files remaining
```

## New Behavior

```
root@mark_1:~# cat /var/log/mycroft/skills.log | grep -i error | grep -i pick
03:55:04.319 - mycroft.skills.core:load_skill:166 - ERROR - Failed to load skill: pick-number.pcwii
PermissionError: [Errno 13] Permission denied: '/opt/mycroft/skills/pick-number.pcwii/settings.json'
03:55:15.213 - mycroft.skills.core:wrapper:1052 - ERROR - An error occurred while processing a request in Pick Number Skill
PermissionError: [Errno 13] Permission denied: '/opt/mycroft/skills/pick-number.pcwii/settings.json'
```
> Disregard the error, I installed the skill as the wrong user originally, but was able to fix that easily

```
root@mark_1:~# tail -f /var/log/mycroft/skills.log
04:37:02.455 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): api.mycroft.ai:443
04:37:02.921 - urllib3.connectionpool - DEBUG - https://api.mycroft.ai:443 "GET /v1/device/0b0cc4df-a286-4bb9-803d-ef1dfe1215d7/skill HTTP/1.1" 304 0
```

## Hardware

Mark 1 Mycroft

### CODE REVIEW

**Comment by KathyReid** [comment]
Thank you so much @sarcasticadmin, these changes are excellent. 

---

To protect yourself, the project, and users of Mycroft technologies we require
a Contributor Licensing Agreement (CLA) before accepting any code contribution.
This agreement makes it crystal clear that along with your code you are
offering a license to use it within the confines of this project. You retain
ownership of the code, this is just a license.

Please visit https://mycroft.ai/cla to initiate this one-time signing. Thank you!

---

Kind regards, 
Kathy

**Comment by KathyReid** [comment]
Changes are 100% correct, just waiting on CLA before merging.

**Comment by sarcasticadmin** [comment]
Thanks @kathyreid! I just initiated the CLA as well.

**Comment by KathyReid** [comment]
Awesome many thanks!

**Comment by penrods** [comment]
I haven't seen a request come through.  Can you either resubmit via https://mycroft.ai/cla or directly send me an email (steve.penrod@mycroft.ai) with your real name and email address so I can send you the digital signature request.

   - Steve

**Comment by KathyReid** [comment]
Checking with @penrods - has CLA been rec'd as yet? 
Best, K.

**Comment by penrods** [comment]
I have not seen a CLA request or email yet.

**Comment by sarcasticadmin** [comment]
@penrods @KathyReid I resubmitted the CLA and emailed @penrods on Tuesday 4/23 but it doesnt sound like either reached you.

I just send out another email with a different email account to @penrods and @KathyReid so hopefully you get that this time around? 🤞

**Comment by sarcasticadmin** [comment]
@kathyreid steve and I were able to figure everything out, CLA signed and I got the confirmation yesterday evening

**Comment by KathyReid** [comment]
@sarcasticadmin awesome. Our process is to wait until the Contributor is added to our [Contributors](https://github.com/MycroftAI/contributors) repo, @penrods is really good about doing this - it's Saturday night-ish where he is so I won't ping him directly until Monday. 

It sounds like a really big process, but it's one-time only - basically in the future if you make any other PRs - and we'd love it if you did! - then I compare your username to the Contributors repo and add a CLA:Yes tag. So it's a bit of a hurdle now, but then we don't need to worry about it for any other PRs in the future. 

TL;DR the changes here are excellent, one minor step to go and then I will merge :smile: 
Again, we really appreciate your efforts!
